gdiff --git a/setup.py b/setup.py
--- a/setup.py	(revision 1f1b703b9e35205946c820e735f58799e1b72d2d)
+++ b/setup.py	(date 1643291122131)
@@ -66,10 +66,10 @@


 def quote_argument(arg):
-    quote = '"' if sys.platform != 'win32' else '\\"'
+    quote = '"'
     return quote + arg + quote

-define_macros = [('MODULE_NAME', quote_argument(PACKAGE_NAME + '.dbapi2'))]
+define_macros = [('MODULE_NAME', quote_argument(f'{PACKAGE_NAME}.dbapi2'))]


 class SystemLibSQLCipherBuilder(build_ext):
@@ -162,9 +162,14 @@
         url="https://github.com/rigglemania/pysqlcipher3",
         package_dir={PACKAGE_NAME: "lib"},
         packages=packages,
+        data_files=[
+            (PACKAGE_NAME, ['sqlcipher.dll', 'libcrypto-1_1-x64.dll'])
+        ],
         ext_modules=[Extension(
             name=PACKAGE_NAME + EXTENSION_MODULE_NAME,
             sources=sources,
+            library_dirs=[r'../sqlcipher'],
+            include_dirs=[r'../'],
             define_macros=define_macros)
         ],
         classifiers=[
